From 63eb1704cbe968be9fc249a510a907a0154101ed Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Tue, 30 Nov 2021 17:39:17 +0000
Subject: [PATCH 1272/2201] Revert "dtoverlays: Drop i2c baudrate for Pi 7inch
 DSI screen to 50kHz"

This reverts commit 0c41710df564f76275c2868beaa0c316553e8247.

The regulator driver is now hopefully fixed, therefore revert the
workaround that dropped the I2C frequency.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts b/arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts
index 5b6f82aaa6fb..5e1700d0367a 100644
--- a/arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts
@@ -95,7 +95,6 @@
 		target = <&i2c0if>;
 		__overlay__ {
 			status = "okay";
-			clock-frequency = <50000>;
 		};
 	};
 
-- 
2.17.1

